mkdocs.yml

```diff
@@ -19,6 +19,7 @@ nav:
   - What is Data Science?: 
     - ... | flat | what_is_data_science/*.md
   - About the team: about.md
+  - PHDInterns: PhDInterns.md
```

Comment:
i think this should come after the articles and projects sections in the structure, but let me know if you dont agree

Reply:
Will Update